src/game/game.cpp

```diff
@@ -650,6 +650,9 @@ bool Game::handle(const SDL_Event &event) {
 
 bool Game::handleMouseButtonDown(const SDL_MouseButtonEvent &event) {
     if (_video) {
+        if (_movieAudio) {
```

Comment:
[Here](https://github.com/seedhartha/reone/blob/f8bceecf3a6fc6b3d8d5bb7473f6771c21fe7a44/src/game/game.cpp#L422) it would be more appropriate to release the audio (so that it gets released when not skipped as well).